### PR TITLE
[test/plugin]support plugin testcases run in docker-compose

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3528 
#3583 

___
### New feature or improvement
- Describe the details and related test reports.

This PR works for complex scenario. For example, SolrJ-Plugin.

### CODE REVIEW

**Comment by dmsolr** [comment]
Configures it in the Jenkins file later

File: test/plugin/runner-helper/src/test/java/org/apache/skywalking/plugin/test/helper/vo/DockerComposeTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+//        FileWriter writer = new FileWriter(writeFile);
+//        yaml.dump(dockerfile, writer);
+//
+//        assertTrue(new File(writeFile).exists());
+//    }
+//
+//    @After
+//    public void tearDown(){
+//        new File(writeFile).deleteOnExit();
+//    }
```
Comment (wu-sheng) [inline test/plugin/runner-helper/src/test/java/org/apache/skywalking/plugin/test/helper/vo/DockerComposeTest.java]: This is unexpected file. Please check others.

Comment (dmsolr) [reply test/plugin/runner-helper/src/test/java/org/apache/skywalking/plugin/test/helper/vo/DockerComposeTest.java]: Got it.

**Comment by wu-sheng** [comment]
Can't find compose case run in this pr, and it didn't show failure. Anything wrong?

**Comment by kezhenxu94** [comment]
> Let’s wait @kezhenxu94 setup 3 tasks to replace parallel across vm.

Set up, Jenkinsfile name `Jenkinsfile-Agent-Test-2`

**Comment by wu-sheng** [comment]
@kezhenxu94 Confirm and merge, please.

**Comment by wu-sheng** [comment]
Merge first, to make other PR continue. @kezhenxu94 any fix could go with a new pr.

**Pull request merged**